File: Easy/Python/1512. Number of Good Pairs.py
class Solution(object):
    def numIdenticalPairs(self, nums):
        # Keep track of number of good pairs
        count = 0
        
        # Go over the array using a nested for loop, comparing
        # each element looking for good pairs
        for i in range(len(nums) - 1):
            for j in range(1, len(nums)):
                if (i < j) and (nums[i] == nums[j]):
                    count += 1
        
        # Return the number of good pairs
        return count

File: Easy/Java/1512. Number of Good Pairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        // Keep track of number of good pairs
        int count = 0;
        
        // Go over the array using a nested for loop, comparing
        // each element looking for good pairs
        for(int i = 0; i < nums.length - 1; i++){
            for(int j = 1; j < nums.length; j++){
                if(i < j && nums[i] == nums[j])
                    count++;
            }
        }
        
        // Return the number of good pairs
        return count;
    }
}
